website/docs/index.html.markdown

```diff
@@ -74,14 +74,38 @@ resource "newrelic_alert_policy_channel" "alert_email" {
 
 The following arguments are supported:
 
-- `api_key` - (Required except for `newrelic_insights_event` resource) Your New Relic API key. The `NEWRELIC_API_KEY` environment variable can also be used.
-- `personal_api_key` - (Required only for the `newrelic_workload` resource) Your New Relic Personal API key. The `NEWRELIC_PERSONAL_API_KEY` environment variable can also be used.
+- `api_key` - (Required) Your New Relic Personal API key. The `NEW_RELIC_API_KEY` environment variable can also be used.
+- `admin_api_key` - (Required for many resources) Your New Relic Admin API key. The `NEW_RELIC_ADMIN_API_KEY` environment variable can also be used.
```

Comment:
Maybe link to the compatibility table here?

Reply:
See https://github.com/terraform-providers/terraform-provider-newrelic/pull/665